src/lists.rs

```diff
@@ -105,7 +120,7 @@ pub fn create_pop_list() -> Result<()> {
 
 pub fn read_pop_list() -> Result<Vec<(String, String)>> {
     let lines = file::read_lines(&pop_path())
-        .chain_err(|| "unable to read pop list. run `cargobomb create-pop-list`?")?;
+        .chain_err(|| "unable to read pop list. run `cargobomb create-full-lists`?")?;
```

Comment:
Yeah. This branch gets rid of all the individual commands, so `create-pop-list` doesn't exist as a separate command anymore, just `create-full-lists`.